packages/angular/projects/ui-angular/src/lib/components/amplify-sign-up/amplify-sign-up.component.html

```diff
@@ -24,6 +24,7 @@
 
 <ng-template #signUpFieldset>
   <fieldset data-amplify-fieldset [disabled]="isPending">
+    <amplify-federated-sign-in></amplify-federated-sign-in>
```

Comment:
This component name shouldn't be opinionated to "sign-in", but I'll address that in another pr after the preview tasks.